pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/SegmentCreationJob.java

```diff
@@ -75,6 +78,9 @@ public SegmentCreationJob(String jobName, Properties properties) throws Exceptio
     _outputDir = _properties.getProperty(PATH_TO_OUTPUT);
     _stagingDir = new File(_outputDir, TEMP).getAbsolutePath();
     _depsJarPath = _properties.getProperty(PATH_TO_DEPS_JAR, null);
+    String readerConfigFilePath = _properties.getProperty(PATH_TO_READER_CONFIG, null);
+    _readerConfig = (null == readerConfigFilePath) ? null
```

Comment:
should we not check the file format type and use the appropriate readerconfig based on filetype?